src/vstest.console/TestPlatformHelpers/TestRequestManager.cs

```diff
@@ -210,7 +196,7 @@ public bool DiscoverTests(DiscoveryRequestPayload discoveryPayload, ITestDiscove
                     ex is SettingsException ||
                     ex is InvalidOperationException)
                 {
-                    LoggerUtilities.RaiseTestRunError(testLoggerManager, null, ex);
+                    ConsoleLogger.RaiseTestRunError(null, ex);
```

Comment:
>ConsoleLogger.RaiseTestRunError(null, ex); [](start = 20, length = 42)

nitpick: can we move this closer to the logger initialization code alone..?

Reply:
Verified that logger initialization is happening within this block only.